### PR TITLE
Remove change email code

### PR DESCRIPTION
**Pull request by mstriemer**
Supports #3049.

### CODE REVIEW

File: src/olympia/users/forms.py [hunk 1]
```diff
@@ -133,6 +133,12 @@ def clean_display_name(self):
         return name
 
 
+USER_EDIT_EXCLUDE_FIELDS = [
+    'password', 'email', 'picture_type', 'last_login', 'fxa_id',
+    'read_dev_agreement',
+]
```
Comment (EnTeQuAk) [inline src/olympia/users/forms.py]: This should be a tuple imho

File: src/olympia/users/tests/test_user_utils.py [hunk 2]
```diff
@@ -5,25 +5,11 @@
 
 from olympia.amo.tests import TestCase
 from olympia.users.models import BlacklistedName, UserProfile
-from olympia.users.utils import (
-    autocreate_username, EmailResetCode, UnsubscribeCode)
+from olympia.users.utils import autocreate_username, UnsubscribeCode
 
 
 class TestEmailResetAndUnsubscribeCode(TestCase):
```
Comment (diox) [inline src/olympia/users/tests/test_user_utils.py]: Change the name of this class as well.

Comment (diox) [inline src/olympia/users/tests/test_user_utils.py]: In fact you can probably make the only method left a function and get rid of the class.

File: src/olympia/users/forms.py (hunk 1 above)
Comment (mstriemer) [inline src/olympia/users/forms.py]: I don't get why it should be a tuple. Semantically it is a list of fields and the positions mean nothing. It is a tuple in some other places though.

Comment (mstriemer) [inline src/olympia/users/forms.py]: I made it a tuple, no big deal either way. Immutability is cool.

**Pull request merged**